api/app/controllers/spree/api/v2/storefront/taxonomies_controller.rb

```diff
@@ -0,0 +1,55 @@
+module Spree
+  module Api
+    module V2
+      module Storefront
+        class TaxonomiesController < ::Spree::Api::V2::BaseController
+          def index
+            render json: serialize(collection), status: 200
+          end
+
+          def show
+            render json: serialize(resource), status: 200
+          end
+
+          private
+
+          def collection
```

Comment:
I think that extracting this to method hurts readability of the code. These are 2 lines of code and are used only once. I'd put this into `index` method What do you think about it?

Reply:
I've done it to make it a little bit more flexible since in case somebody wants to override it all they have to do is to change `collection` & `resource` methods instead of whole controller action. I think it'd be cool for other developers to have a possibility of changing basically every component without a need to replace entire methods